daal4py/sklearn/test/test_run_to_run_stability_tests.py

```diff
@@ -14,322 +14,334 @@
 # limitations under the License.
 #===============================================================================
 
-import unittest
-import numpy as np
 import daal4py as d4p
+import numpy as np
+import pytest
 
 from daal4py.sklearn import patch_sklearn
 patch_sklearn()
 
-# to reproduce errors even in CI
-d4p.daalinit(nthreads=100)
-
-from sklearn.ensemble import RandomForestClassifier
-from sklearn.ensemble import RandomForestRegressor
-from sklearn.linear_model import LogisticRegression
-from sklearn.linear_model import LogisticRegressionCV
-from sklearn.neighbors import KNeighborsClassifier
-from sklearn.linear_model import LinearRegression
-from sklearn.linear_model import Ridge
-from sklearn.linear_model import ElasticNet
-from sklearn.linear_model import Lasso
-from sklearn.cluster import KMeans
-from sklearn.cluster import DBSCAN
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.linear_model import LogisticRegression, LogisticRegressionCV
+from sklearn.neighbors import KNeighborsClassifier, KNeighborsRegressor, NearestNeighbors
+from sklearn.linear_model import LinearRegression, Ridge, ElasticNet, Lasso
+from sklearn.cluster import KMeans, DBSCAN
 from sklearn.decomposition import PCA
-
 from sklearn.svm import SVC
-from sklearn.datasets import make_classification
-from sklearn.datasets import make_regression
-
-##################################################
+from sklearn.manifold import TSNE
+from sklearn.model_selection import train_test_split
 
-def run_clf_test(func):
-    for features in [5, 10]:
-        X, y = make_classification(n_samples=4000, n_features=features, n_informative=features, n_redundant=0,
-            n_clusters_per_class=8, random_state=0)
+from sklearn.datasets import (make_classification, make_blobs,
+                              load_iris, load_boston)
+from sklearn.metrics import pairwise_distances
+from scipy import sparse
 
-        baseline, name = func(X, y)
 
-        for i in range(10):
-            res, _ = func(X, y)
+# to reproduce errors even in CI
+d4p.daalinit(nthreads=20)

```

Comment:
There was 100, why did it decrease?

Reply:
I changed it when I was doing local testing and forget correct.